libs/minecraft/PosUtils.arucas

```diff
@@ -45,26 +60,54 @@ enum Direction {
         return Direction.NONE;
     }
 
-    fun vec() {
+    fun rotateYClockwise() {
+        return this.right();
+    }
+
+    fun rotateYCounterclockwise() {
+        return this.left();
+    }
+
+    fun getAxis(){
+        return this.axis;
+    }
+
+    fun asVector() {
         return this.vector.copy();
     }
 
-    fun x() {
+    fun unitVec(){
```

Comment:
Do we need `unitVec` and `asVector`, I'd rather just keep one: `asVector`

Reply:
(Same as mult) I think it's more confusing and `unitVec` should be removed so we just keep `asVector`